terra_layer/migrations/0051_auto_20201014_1348.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.1.2 on 2020-10-14 13:48
```

Comment:
Why it seems to be an automated migration (see file name) but it's a manual data migration

Reply:
I create an empty migration with `python manage.py makemigrations --empty`. I followed the documentation for data migration https://docs.djangoproject.com/fr/2.2/topics/migrations/#data-migrations